src/types/my-cooperations/enums/myCooperations.enums.ts

```diff
@@ -0,0 +1,10 @@
+export enum ResourcesAvailabilityEnum {
+  openAll = 'openAll',
+  openManually = 'openManually'
+}
```

Comment:
keep in mind convention for enum naming

<img width="725" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/3e787360-ec75-4753-8cad-6538a2b75529">